README.rst

```diff
@@ -60,15 +60,15 @@ Requirements
 
  * `pass <http://www.passwordstore.org/>`_
  * `xclip <http://sourceforge.net/projects/xclip>`_
- * `python <https://www.python.org/>`_
+ * `python 3 <https://www.python.org/>`_
```

Comment:
I failed to test this with python3.2, but 3.3 works fine, so that makes sense. I won't honestly be providing any help for 3.2 anyway, so that also makes sense.